test/remote/gateways/remote_litle_certification_test.rb

```diff
@@ -803,6 +822,25 @@ def sale_assertions(amount, card, options, assertions={})
     assert_equal 'Approved', response.message
   end
 
+  def three_d_secure_assertions(test_id, card, type, source, result)
+    credit_card = CreditCard.new(:number => card, :month => '01',
+                                 :year => '2021', :brand => type,
+                                 :verification_value => '261',
+                                 :name => 'Mike J. Hammer')
+
+    options = {
+      order_id: test_id,
+      order_source: source,
+      cavv: 'BwABBJQ1AgAAAAAgJDUCAAAAAAA=',
+      xid: 'BwABBJQ1AgAAAAAgJDUCAAAAAAA='
+    }
+
+    assert response = @gateway.authorize(10100, credit_card, options)
+    assert_success response
+    assert_equal result, response.params['fraudResult_authenticationResult']
+    puts "Test #{test_id}: #{txn_id(response)}"
```

Comment:
This isn't just for debugging, I added these `puts`es to aid in certification - running the cert test will output all the txn_ids needed to provide to Vantiv and which test they're associated with. It's conceivable this would need to be done again in the future.